tests/poms/inventory.page.ts

```diff
@@ -19,15 +26,56 @@ export namespace InventoryService {
 		async close() {
 			await this.page.close();
 		}
+
+		async addProductToCart(addProductButtonTestId: string) {
+			const addProductButton = this.page.locator(
+				`[data-test="${addProductButtonTestId}"]`
```

Comment:
should use the locator that is visible to user here. like innerText. and I think the realworld wouldn't have this kind of luxury attribute like data-test for every fetched products possible.